lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+
+  attr_accessor :cells
+
+  def reset!
+    @cells=[" "," "," "," "," "," "," "," "," "]
+  end
+
+  def initialize
+    reset!
+  end
+
+  def display
+    puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts "-----------"
+    puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts "-----------"
+    puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(index)
+    @cells[index.to_i - 1]
+  end
+
+  def full?
+    @cells.include?(" ") ? false : true
+  end
```

Comment:
Avoid using the ternary operator to return `true` and `false`. Instead, rely on the implicit boolean return value of the expression. 

`cells.none? {|cell| cell == " " || cell == nil}`